client/src/pages/Admin/Publications.vue

```diff
@@ -14,18 +14,18 @@
         >
           <q-input
             v-model="new_publication.name"
+            :error="$v.new_publication.$error"
             outlined
             label="Enter Name"
             data-cy="new_publication_input"
-          />
-          <q-banner
-            v-if="tryCatchError"
-            dense
-            rounded
-            class="form-error text-white bg-negative text-center q-mt-xs"
-            data-cy="banner_form_error"
-            v-text="$t(`publications.create.failure`)"
-          />
+          >
+            <error-field-renderer
+              :errors="$v.new_publication.name.$errors"
+              prefix="publications.create.name"
+              data-cy="name_field_error"
+            />
```

Comment:
It seems the convention in Quasar by default is to display one error message at a time. I'm curious what the reasoning is to deviate from that and show multiple error messages at once. I can imagine there's advantages to both approaches but I don't think I fully understand all of the reasoning behind this choice.

Also, all of the [examples of QInputs with nested content in Quasar's documentation](https://quasar.dev/vue-components/input) shows them using a `<template>` tag with a `slot` attribute  to dictate the position of the nested content relative to the input.

```suggestion
            <template #append>
              <error-field-renderer
                :errors="$v.new_publication.name.$errors"
                prefix="publications.create.name"
                data-cy="name_field_error"
              />
            </template>
```

Reply:
Quasar has no convention.  It just gives you a slot that you can fill with whatever you like.  Vuelidate evaluates all of the rules independently so each field could have more than one error.  Seems better to show them to the user (if they all fit comfortably) than to try to pick which error to show them.

And yes, the error field renderer should be in a slot.  I'll refactor.